src/models/local/LocalBaseController.js

```diff
@@ -92,6 +97,27 @@ export default class LocalBaseController {
     return !!abi.find(fn => fn.type === "constructor");
   }
 
+  hasSelfDestruct(contractDataPath) {
+    if (!fs.exists(contractDataPath)) return false
+    const ast = fs.parseJson(contractDataPath).ast
+    if (this.hasKeyValue(ast, "typeIdentifier", "t_function_selfdestruct_nonpayable$_t_address_$returns$__$")) return true
+    for (let i = 0; i < ast.nodes.length; i++) {
+      for (let j = 0; j < (ast.nodes.baseContracts || []).length; j++) {
```

Comment:
Could you change these for-loops to use either the [`for-of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) construct for [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) method, for better readability?